WordPress/src/main/java/org/wordpress/android/ui/plans/PlanFragment.java

```diff
@@ -73,17 +72,16 @@ public void onSaveInstanceState(Bundle outState) {
     }
 
     private void refreshPlanUI() {
```

Comment:
Hey @nbradbury , with `refreshPlanUI()` called only in `onActivityCreated()` now, its function is more like setting up some View instead or refreshing it. It might be useful to rename it accordingly to avoid confusion. Thanks!